pkg/webui/console/containers/applications-table/index.js

```diff
@@ -79,8 +111,13 @@ export default class ApplicationsTable extends Component {
         getItemsAction={this.getApplicationsList}
         baseDataSelector={this.baseDataSelector}
         searchable
-        {...this.props}
+        tabs={isAdmin ? tabs : []}
+        {...rest}
       />
     )
   }
 }
+
+export default connect(state => ({
+  isAdmin: selectUser(state).isAdmin || false,
```

Comment:
```suggestion
  isAdmin: selectUserIsAdmin(state),
```

Use the `selectUserIsAdmin` selector.